### ISSUE TITLE
NTR: Impaired ability to shift

### ISSUE THREAD

**Issue by nicolevasilevsky** [obophenotype/human-phenotype-ontology#6053]
**Preferred term label:**
Impaired ability to shift

**Synonyms**
Sticky behavior
Inflexible behavior - broad
Impaired set shifting

**Definition (free text, please give PubMed ID)**
A behavioral abnormality whereby a person is unable to move freely from one situation, activity or aspect of aa problem to another as circumstances demand. Key aspects of shifting include the ability to make transitions, problem solve flexibly, switch or alternate attention, and change focus from one mind-set or topic to another. Mild deficits in the ability to shift compromise the efficiency of problem solving, whereas more severe difficulties are reflected in perseverative behaviors. Sometimes this is described as being rigid or inflexible.
Source:  _updated_ this is from the BRIEF manual,  noted below.

_Here is an alternative definition:_
An impairment in executive functioning whereby there is an inability to modify attention and behavior in response to changing circumstances and demands.
Source: PMID:26039846

**Parent term (use hpo.jax.org/app)**
Impaired executive functioning

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Autism spectrum disorder

This is needed fo the Kids First INCLUDE supplement.

**Comment by pnrobinson** [comment]
@nicolevasilevsky Can you ask Stephanie for a more precise label. To shift could be interpreted as shifting a physical object. Also, please let's try to get PMIDs for references!

**Comment by nicolevasilevsky** [comment]
Stephanie checked with an expert Lisa Jacobson, and Lisa recommends “inflexibility” or “sticky” as the primary label.

I asked Stephanie to ask Lisa for a citation for the reference, she is out of office until Aug 09, so we may not hear back right away.

Thanks!

**Comment by nicolevasilevsky** [comment]
@pnrobinson this is the publication info for the BRIEF2 manual. Not sure how to reference this in Protege?

![image](https://user-images.githubusercontent.com/6722114/93613808-30ed1880-f986-11ea-98a5-9bf1d8d23b0d.png)

![image](https://user-images.githubusercontent.com/6722114/93613901-4f531400-f986-11ea-9857-14b01fb95783.png)

https://www.parinc.com/Products/Pkey/4523

**Comment by pnrobinson** [comment]
If there is no other way, them we can use an ISBN, but this is not that useful as a reference since it cannot be accessed by most users. We should try to find a PMID!

**Comment by nicolevasilevsky** [comment]
I updated the suggested def above, with a PMID

**Comment by pnrobinson** [comment]
I would suggest we use the above PMID but with the label "Inability to shift focus"

@bcoleman96  thoughts?

**Comment by nicolevasilevsky** [comment]
I like this label suggestion

**Comment by bcoleman96** [comment]
I agree. "Inability to shift attention" might be better since set-shifting might be considered an inability to focus. 

In terms of the definition, have we already had a discussion about set shifting vs cognitive shifting? There seems to be some debate on whether these are different things or just often confused.

I would also suggest "impaired task shifting" or "inability to task shift" as synonyms.

**Comment by pnrobinson** [comment]
adding this term with this ref
https://pubmed.ncbi.nlm.nih.gov/23356593/

label **Impaired ability to shift attention**
